parser-typechecker/src/Unison/TermPrinter.hs

```diff
@@ -891,11 +889,14 @@ calcImports im tm = (im', render $ getUses result)
     --   - is > 1, or
     --   - is 1, and S is an infix operator.
     -- Also drop names with an empty prefix.
+    lookupOrDie s m = fromMaybe msg (Map.lookup s m) where
+      msg = error $ "TermPrinter.enoughUsages " <> show (s, m)
+
     enoughUsages :: Map Suffix (Prefix, Int) -> Map Suffix (Prefix, Int)
-    enoughUsages m = (Map.keys m) |> filter (\s -> let (p, i) = fromJust $ Map.lookup s m
+    enoughUsages m = (Map.keys m) |> filter (\s -> let (p, i) = lookupOrDie s m
```

Comment:
Getting rid of `fromJust` in favor of actual error message.

Reply:
better than crashing with no message :)